source/manual/deployment_dashboards.html.md

```diff
@@ -0,0 +1,190 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-05-01
+title: Deployment Dashboards
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+# Deployment Dashboards
+
+## Introduction
+
+There are a number of applications with a dashboard showing useful information for the deployment process.
+
+The existing deployment dashboards are written by puppet every 30 minutes and loaded when Grafana starts. Don’t change them directly.
+
+## Panels
+
+### Deployment and restart lines
+
+These show/hide vertical lines on the other panels that show when events happened.
+
+![Deployment restart lines](images/deployment_dashboards/deploys_restarts_checkboxes.png)
+
+![Deployment lines](images/deployment_dashboards/deployment_lines.png)
+
+### Processes - Last hour
+
+![Processes](images/deployment_dashboards/processes.png)
+
+This shows the number of processes running on each server. During a deployment, you should expect the number of processes to double and then return to normal (the number depends on the server).
+
+You should only test your code when this has settled back to normal.
+
+Implementation note
```

Comment:
The linebreak after this line doesn't appear on the page. Maybe join lines 36 and 37 and add a colon after "Implementation note"?